src/main.cpp

```diff
@@ -78,9 +78,10 @@ int main(int argc, char** argv) {
     if (cli->has("list-monitors")) {
       for (auto&& mon : randr_util::get_monitors(conn, conn.root(), true)) {
         if (WITH_XRANDR_MONITORS && mon->output == XCB_NONE) {
-          printf("%s: %ix%i+%i+%i (XRandR monitor)\n", mon->name.c_str(), mon->w, mon->h, mon->x, mon->y);
+          printf("%s: %ix%i+%i+%i (XRandR monitor)%s\n", mon->name.c_str(), mon->w, mon->h, mon->x, mon->y,
```

Comment:
I think it would look better if primary monitors here were marked with `(XRandR monitor, primary)` instead of `(XRandR monitor) (primary)`

```suggestion
          printf("%s: %ix%i+%i+%i (XRandR monitor%s)\n", mon->name.c_str(), mon->w, mon->h, mon->x, mon->y,
```